src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -44,11 +44,14 @@ end = struct
   module Scan_state = Transaction_snark_scan_state.Make (Inputs)
 
   module Staged_ledger_error = struct
+    type partition = int * int option [@@deriving sexp]
```

Comment:
maybe this should be a record rather than a tuple, so the two ints can be labeled

Reply:
This field is mainly for debugging because there is no concept of a partition outside `Staged_ledger`. But since it is exposed, I guess it makes sense to add labels